PokeOneWeb.Shared/Extensions/StringExtensions.cs

```diff
@@ -6,5 +6,10 @@ public static bool EqualsExact(this string str, string other)
         {
             return str.Equals(other, StringComparison.Ordinal);
         }
+
+        public static bool EqualsExact2(this string str, string other)
```

Comment:
Why do you need this duplicate?

Reply:
Removed, was for testing purpose for the code coverage on new lines only.